### PR TITLE
Remove temporal paths left out during training.

### PR DESCRIPTION
**Pull request by franchuterivera**
Addresses #110

### CODE REVIEW

**Review by ravinkohli — APPROVED** [review]
Thanks for this PR. I think it can be merged

File: autoPyTorch/pipeline/components/training/trainer/__init__.py [hunk 1]
```diff
@@ -384,12 +395,6 @@ def early_stop_handler(self, X: Dict[str, Any]) -> bool:
             torch.save(X['network'].state_dict(), best_path)
 
         if epochs_since_best > X['early_stopping']:
```
Comment (nabenabe0928) [inline autoPyTorch/pipeline/components/training/trainer/__init__.py:397]: `return epochs_since_best > X['early_stopping']`

**Review by nabenabe0928 — CHANGES_REQUESTED** [review]
Thanks for the PR.
I suggested one minor change.

File: autoPyTorch/pipeline/components/training/trainer/__init__.py (hunk 1 above)
Comment (franchuterivera) [reply autoPyTorch/pipeline/components/training/trainer/__init__.py]: Thanks for the suggestion, I have incorporated this.

**Pull request closed without merging**